src/installation/guides/fde.md

```diff
@@ -260,8 +260,13 @@ install_items+=" /boot/volume.key /etc/crypttab "
 
 Next, install the boot loader to the disk.
 
+For legacy BIOS systems:
+```
+# grub-install --target=i386-pc /dev/vda
+```
+For UEFI systems:
 ```
-# grub-install /dev/sda
+# grub-install --target=x86_64-efi --efi-directory=/efi --bootloader-id="Void"
```

Comment:
```suggestion
# grub-install --target=x86_64-efi --bootloader-id="Void"
```
I assume it isn't necessary to specify a path to `/boot/efi` and I recommend against the change from `/boot/efi` to `/efi`. However, I don't know anything about GRUB, so `--efi-directory=/boot/efi` may still be required.

Reply:
This EFI directory is changed from /boot/efi to /efi to prevent a [Evil Maid](https://en.wikipedia.org/wiki/Evil_maid_attack) attack on the /boot partition (which would be encrypted if the EFI directory is /efi)

Reply:
The disadvantage is that you have to specify the alternative path for GRUB.

The possibility of an Evil Maid attack has nothing to do with the mount point of the EFI system partition. You add no defense by mounting it somewhere else.

Reply:
In this guide, `/boot` is encrypted already, so I'm not sure what you mean.  ESP is going to be subject to Evil Maid no matter where it gets mounted, since it'll never be encrypted.